src/main/java/com/user/management/data/user/dao/impl/UserDaoImpl.java

```diff
@@ -15,88 +16,81 @@
 
 /**
  * An implementation class of the User Data Access Object.
- * */
-
+ */
 public class UserDaoImpl implements UserDao {
 
     public static final Logger LOGGER = LoggerFactory.getLogger(UserDaoImpl.class);
+
     @Override
-    public User findUserByUsername(String username){
-        String selectByIdQuery = "SELECT * FROM login WHERE username=?";
-        User User = null;
+    public User findUserByUsername(String username) {
+        User user = null;
 
         try (Connection connection = ConnectionHelper.getConnection();
```

Comment:
fix the try-with-resource. remove preparedstatement from try()

Reply:
okay napo sir